packages/api-gen/src/patchJsonSchema.ts

```diff
@@ -20,7 +20,7 @@ export const extendedDefu = createDefu((obj, key, value) => {
   if (Array.isArray(obj[key]) && Array.isArray(value)) {
     // concat arrays but skip duplicates
     // @ts-expect-error - we know that obj[key] is an array as we're inside this if statement
-    obj[key] = [...new Set([...obj[key], ...value])];
+    obj[key] = [...new Set([...obj[key], ...value])].sort();
```

Comment:
Why is it important to sort it out?

Reply:
for overriding - depending on which source came first it was looking random, also snapshots often changed when I experimented with tests, this way we keep consistency